Sources/MockoloFramework/Operations/OutputWriter.swift

```diff
@@ -37,8 +38,21 @@ func write(candidates: [(String, Int64)],
         break
     }
 
-    let importsSet = Set(importLines.map{$0.trimmingCharacters(in: .whitespaces)})
-    let importLineStr = importsSet.sorted().joined(separator: "\n")
+    var imports = importLines.compactMap { (importLine) -> String? in
```

Comment:
Add a check if testableimports is nil or empty, and do this only if not; else it's converting to and from module and import lines redundantly.